tools/importers/SynchronizeTypographyTokens.spec.ts

```diff
@@ -0,0 +1,80 @@
+import {
+	ImportTypographyVariables,
+	ImportTypographyTokens,
+	requestForBin,
+} from './SynchronizeTypographyTokens';
+const fs = require('fs');
+const tokensFilesConfig = require('./configs/FakeSynchronizeTypographyTokensConfig.json');
```

Comment:
do we need to keep config in separate file? Can't we define it in test file?

Reply:
It is needed now in testes, but if I will remove it from tests, this export will be redundant.

Reply:
> it isn't good practice to change API for tests.
> Tests should try to use code in the same way as we do in production code.

I agree, but if code was written completely without thinking how to use it in tests, or it was not split enought - it should be changed.